omegatrade-app/frontend/src/app/components/simulation/simulation.component.html

```diff
@@ -0,0 +1,119 @@
+              <mat-select formControlName="data" id="data">
+                <mat-option *ngFor="let record of noOfRecords" [value]="record">{{ record }}</mat-option>
+              </mat-select>
+            </mat-form-field>
+          </div>
+          <div class="col-1 mat-actions ">
+            <button type="submit" class="start-simulate-btn" matTooltip="Start Simulation" mat-stroked-button
+              [disabled]="!simulateForm.valid">Simulate</button>
+          </div>
+
+        </div>
+      </form>
+    </mat-card>
+  </div>
+
+  <div class="container-fluid mt-20">
+    <div class="row">
+      <div class="col-6 simulate-list-section">
+        <span class="simulate-list-title">
+          Manage Simulations
+        </span>
+      </div>
+      <div class="col-6 table-options">
+        <mat-form-field>
+          <input matInput [(ngModel)]="searchInput" (ngModelChange)="applyFilter()" placeholder="Search">
+        </mat-form-field>
+      </div>
+    </div>
+    <div class="row mt-10">
+      <div class="col-12">
+        <table mat-table [dataSource]="dataSource" matSort class="mat-elevation-z8">
+
+          <!-- Name Column -->
+          <ng-container matColumnDef="companyName">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header> Company Name </th>
+            <td mat-cell *matCellDef="let simulation"> {{simulation.companyName}} </td>
+          </ng-container>
+
+          <!-- shortcode Column -->
+          <ng-container matColumnDef="companyShortCode">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header> Short Code </th>
+            <td mat-cell *matCellDef="let simulation"> {{simulation.companyShortCode}} </td>
+          </ng-container>
+
+          <ng-container matColumnDef="status">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header>Simulation Status </th>
+            <td mat-cell *matCellDef="let simulation"> {{simulation.status}} </td>
+          </ng-container>
+
+          <ng-container matColumnDef="action">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header> Action</th>
+            <td mat-cell *matCellDef="let row" class="action-button">
+              <a href="javascript:void(0)" class="dashboard-redirect-btn" matTooltip="view simulation" [routerLink]="['/dashboard']" [queryParams]="{companyId: row.companyId}" ><i class="material-icons">visibility</i></a>
+                &nbsp;&nbsp;
+              <a href="javascript:void(0)" class="delete-company-btn" matTooltip="delete simulation"
+                (click)="deleteSimulation(row)"><i class="material-icons">delete</i></a>
+                &nbsp;&nbsp;
+              <a href="javascript:void(0)" matTooltip="start simulation" *ngIf="row.status =='STOPPED'" class="start"
+                (click)="updateSimulation(row.sId,row.companyId,'PROCESSING')">
+                <mat-icon aria-hidden="false" aria-label="Example home icon">play_circle</mat-icon>
```

Comment:
nit: `Example home icon` seems to be a copy-paste error

Reply:
the above issue has been resolved